www/schemas/BHHSAgentAdditionalProperties.yaml

```diff
@@ -0,0 +1,41 @@
+type: object
+description: BHHS agent additional properties
+properties:
+  type:
+    type: string
+    description: BHHS agent additional properties
+    enum:
+      - BHHSAgentAdditionalProperties
+      
+  affiliateId:
+    type: string
+    example: CA301
+    
+  officeId:
+    type: string
+    example: CA301-001
+    
+  employeeType:
+    type: string
+    example: Full Time Sales Professional
+      
+  status:
+    description: the status of object
+    type: string
+    example: Active
+
+  isDisplayed:
```

Comment:
isDisplayed is a bad attribute name - is displayed where?  I recommend renaming this to isDisplayedOnCompanyWebsite (boolean) or a more generic displayedOnWebsite: (array of website urls)